### DISCUSSION TITLE
math mode macro doesn't trigger

### DISCUSSION THREAD

**Discussion by luciandf** [texstudio-org/texstudio#1728] [General]
hello all,

I have been using the following math mode macros for subscript and superscript which was working great until recently when it doesn't trigger anymore. Here is the macro:
Subscript
`_{%|}`
`(?highlighted-as:numbers,math-delimiter,math-keyword)_`

Superscript
`^{%|}`
`(?highlighted-as:numbers,math-delimiter,math-keyword)\^`

has anything changed?

**Comment by Gomesz785** [comment]
Did you changed the directories or updated the texstudio or did something major on your computer?  

If not please start a issue. And provide the information stated in the issue-boilerplate. It will be most helpful to other lads who keep this running!

😊

**Reply by luciandf** [reply]
the only thing I did was to update texstudio whenever it announced a new version.

**Comment by sunderme** [comment]
functionality has changed in txs and I have not yet had an idea on how to recreate it.
